torch/onnx/symbolic_opset9.py

```diff
@@ -1612,3 +1612,109 @@ def gather(g, self, dim, index, sparse_grad=False):
     index = g.op("Cast", g.op("OneHot", index, depth, values, axis_i=dim), to_i=sym_help.cast_pytorch_to_onnx[dtype])
     mul = g.op("Mul", g.op("Unsqueeze", self, axes_i=[dim + 1]), index)
     return g.op("ReduceSum", mul, axes_i=[dim], keepdims_i=0)
+
+
+def index(g, self, index):
+    if sym_help._is_packed_list(index):
+        indices = sym_help._unpack_list(index)
+    else:
+        indices = [index]
+
+    def try_mask_to_index(index):
+        if not index.node().mustBeNone() and index.type().scalarType() == "Byte":
+            warnings.warn("Exporting aten::index operator with indices of type Byte. "
+                          "Only 1-D indices are supported. In any other case, "
+                          "this will produce an incorrect ONNX graph.")
+            index = squeeze(g, nonzero(g, index), dim=1)
```

Comment:
also seems our unittest didn't trigger the Nonzero part. Otherwise, the ci shoudl fail.

Reply:
Good catch, adds a test case that uses nonzero.